packages/opentelemetry/lib/tracing/serverInstrumenation.js

```diff
@@ -0,0 +1,86 @@
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
+
+class ServerInstrumentation {
+  constructor () {
+    this._httpInstrumentation = new HttpInstrumentation();
+    this._expressInstrumentation = new ExpressInstrumentation();
+    this._ignoreIncomingPaths = [],
+    this._ignoreOutgoingUrls = [],
+    this._httpCurrentConfig = {
+      ignoreIncomingPaths: this._ignoreIncomingPaths,
+      ignoreOutgoingUrls: this._ignoreOutgoingUrls,
+      applyCustomAttributesOnSpan: {},
+      requestHook: undefined,
```

Comment:
this is confusing. `undefined` means no value. I would rather use null or remove it from the object at all